app/routes/events_.$eventId.tsx

```diff
@@ -73,9 +88,14 @@ export default function EventRoute() {
             return <li key={user.id}>{user.name}</li>;
           })}
         </ul>
-        <button className="w-full md:px-[30px] md:py-[15px] md:w-max bg-warm rounded-full py-4 text-white">
-          Join Event
-        </button>
+        <Form method="post">
+          <button
+            className="w-full md:px-[30px] md:py-[15px] md:w-max bg-warm rounded-full py-4 text-white"
+            type="submit"
```

Comment:
Should probably disappear if already joined!

Reply:
will do it in a seperate PR :D

Reply:
Captured in issue https://github.com/social-plan-it/plan-it-social-web/issues/137